impl/src/main/java/io/kroxylicious/testing/kafka/common/KafkaClusterConfig.java

```diff
@@ -147,13 +147,19 @@ public static KafkaClusterConfig fromConstraints(List<Annotation> annotations) {
                     throw new RuntimeException(e);
                 }
             }
-            if (annotation instanceof SaslPlainAuth) {
+            if (annotation instanceof SaslPlainAuth.List) {
```

Comment:
you can use JEP 305 (instanceof pattern matching)

Reply:
Done